src/xenserver/opt/xensource/www/XCP-ng-index.html

```diff
@@ -316,7 +316,7 @@ <h3 class="subtitle">Management tools</h3>
         <div class="col-md">
           <div class="box">
             <h4>Xen Orchestra</h4>
-            <a href="#ex1" rel="modal:open"><img class="preview" src="./asset/xoa.jpg" /></a>
+            <a href="#ex1" rel="modal:open"><img class="preview" src="./asset/xoa.jpg" alt="Open XOA" /></a>
```

Comment:
This does not look related to this PR and is not mentioned in any commit messages.

Reply:
done